config/database.yml

```diff
@@ -1,17 +1,64 @@
-default: &default
+# PostgreSQL. Versions 8.2 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On OS X with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On OS X with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem 'pg'
+#
+development:
   adapter: postgresql
   encoding: unicode
+  database: cfp_app_development
```

Comment:
Hm, if this file isn't in the .gitignore, we should add it. Please remove the changes to this file.